src/opossum_lib/opossum_file.py

```diff
@@ -60,32 +61,74 @@ class Metadata:
     buildDate: Optional[str] = None
 
 
+class ResourceType(Enum):
+    FILE = auto()
+    FOLDER = auto()
+    TOP_LEVEL = auto()
+    OTHER = auto()
```

Comment:
what is `OTHER`?

Reply:
Coming from the conversion of an SPDX document I wanted to use this type for all elements that are neither a file nor a folder, so for example the relationship type or a reference to an external document.